### PR TITLE
Modify listener interface in meta client

### PR DESCRIPTION
**Pull request by critical27**
Modify listener interface in meta client, needed by https://github.com/vesoft-inc/nebula-storage/pull/180.

### CODE REVIEW

**Review by bright-starry-sky — DISMISSED** [review]
LGTM. Good job.

**Comment by critical27** [comment]
Hold on, there could be more changes as for PartManager

**Comment by critical27** [comment]
Ready

**Review by bright-starry-sky — DISMISSED** [review]
LGTM

**Comment by critical27** [comment]
Ready, verified.

**Review by bright-starry-sky — DISMISSED** [review]

**Comment by bright-starry-sky** [comment]
LGTM.

**Review by darionyaphet — APPROVED** [review]

File: src/common/meta/Common.h [hunk 1]
```diff
@@ -29,7 +30,34 @@ struct PartHosts {
...
+    std::vector<HostAddr>   peers_;
+};
+
 using PartsMap = std::unordered_map<GraphSpaceID, std::unordered_map<PartitionID, PartHosts>>;
+using ListenersMap =
+    std::unordered_map<GraphSpaceID, std::unordered_map<PartitionID, std::vector<ListenerHosts>>>;
+using RemoteListenerInfo = std::pair<HostAddr, cpp2::ListenerType>;
+using RemoteListeners =
+    std::unordered_map<GraphSpaceID,
+                       std::unordered_map<PartitionID, std::vector<RemoteListenerInfo>>>;
```
Comment (panda-sheep) [inline src/common/meta/Common.h:60]: It is best to add comments  about `ListenersMap ` and `RemoteListeners`.

**Review by panda-sheep — COMMENTED** [review]
Good job，Excellent！

File: src/common/meta/Common.h (hunk 1 above)
Comment (critical27) [reply src/common/meta/Common.h:64]: Yeah, good suggestion, done.

**Review by bright-starry-sky — APPROVED** [review]
Well done.

**Review by panda-sheep — APPROVED** [review]
Excellent!

**Pull request merged**